src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from "./Breadcrumb.module.css"
+
+export interface LinkProps {
+  href: string
+  label: string
+  isCurrent?: boolean
+}
```

Comment:
As I mentioned previously, this is what I envision happening here.
```suggestion
export interface BreadcrumbItemProps {
  link: Link
  isCurrent: boolean
}
```